docs/explanations/introduction.md

```diff
@@ -0,0 +1,29 @@
+
+The main intended use case is to secure proof-of-work (PoW) blockchains against 51% attacks with checkpointing, as described in [this paper](https://iohk.io/en/research/library/papers/securing-proof-of-work-ledgers-via-checkpointing/). More specifically to secure the ETC chain, as proposed in [ECIP1097](https://ecips.ethereumclassic.org/ECIPs/ecip-1097), whose description of a checkpointing federation Morpho is based on. While the implementation has specifics to this use case, there is very little preventing it to be more generic.
+
+## Checkpointing and 51% attacks
+
+51% attacks are possible when an adversary has more than half of the hashing power in the entire network, meaning they can produce blocks at a faster rate than all other miners combined. Over time this allows them to create a forking chain in private which is arbitrarily longer than the longest public chain. At any point the attacker can then disclose their private chain to the public, at which point everybody will switch to the attackers chain, since it's longer. This is a problem because the attacker can choose which transactions are included in their chain. As an example, double spending is possible by not including an already considered confirmed transaction from the public chain in their private chain.
+
+An important aspect to consider is that 51% attacks become easier with lower mining participation, since an attacker only needs more mining power than the entirety of the lawful miners. Because of this, unpopular networks are at an increased risk, which can especially happen when they're new, or towards the end of their lifetime.
+
+Checkpointing is a proposed mechanism to solve this problem, securing a proof-of-work chain against 51% attacks, even with low mining participation. The main idea is to use a small number of trusted community members, a checkpointing federation, to regularly insert _checkpoints_ into the chain. Checkpoints are special non-proof-of-work blocks that _cannot_ be reverted, essentially confirming all transactions in preceding blocks. Instead of always preferring the longest chain, network nodes should then always adopt the chain with the latest checkpoint. In order to ensure that checkpoint blocks can only be created by the checkpointing federation, the blocks include a list of member signatures signing the preceding block hash. The honest nodes will have to be aware of each members public key in order to verify such blocks. For more details, see especially section 3 of [the paper](https://iohk.io/en/research/library/papers/securing-proof-of-work-ledgers-via-checkpointing/), or [the video discussing it](https://www.youtube.com/watch?v=aasUIB1W81E).
```

Comment:
members public key -> member's public key
[the paper] -> [the checkpointing paper]